src/Kdyby/Console/DI/ConsoleExtension.php

```diff
@@ -82,18 +82,16 @@ public function loadConfiguration()
 		Nette\Utils\Validators::assert($config, 'array');
 		foreach ($config['commands'] as $i => $command) {
 			$def = $builder->addDefinition($this->prefix('command.' . $i));
-			list($def->factory) = Nette\DI\Compiler::filterArguments(array(
-				is_string($command) ? new Statement($command) : $command
-			));
-
-			if (class_exists($def->factory->entity)) {
-				$def->class = $def->factory->entity;
-			}
-
-			$def->setAutowired(FALSE);
-			$def->setInject(FALSE);
```

Comment:
Wait, isn't the service graph initialized only when parsing `config.neon` and generating the container? The injects should be part of the generated code, shouldn't they?

If it's during the generation only, it doesn't matter if the graph is or isn't init'ed -- it should be in fact better this way, as the container code will match the use-case of your command more exactly.